CHANGELOG.md

```diff
@@ -6,6 +6,40 @@ in this file.
 Detailed changes to the Solar Forecast Arbiter Core python library can be found
 in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
 
+## [1.0beta4] - Unreleased
+
+### Added
+
+- Report data may now be filtered by quality flag. Currently allows
+  filtering of *user flagged* and *nighttime* values.
+
+- Reports now contain a summary of data affected by the data resampling and
+  alignment process. The summary includes the number of data points removed by
+  each phase of validation.
+
+- Reports now contain a table of all included metrics over the entire selected
+  period.
+
+- Reports in the *pending* and *failed* state now display a message to the user
+  about their status. For failed reports, this is a list of errors encountered
+  while processing the report.
+
+- The dashboard report view now allows users to search metric plots by
+  forecast ant to sort metric plots by metric, category and forecast.
+
+- Dashboard users can now download report metrics as a csv.
```

Comment:
is this not accessible through the public api?

Reply:
The metrics are available as part of the processed `raw_report` through the api, but not as a csv. The csv is generated through some javascript that Tony introduced that converts the metrics cds to a csv.

Reply:
hmmm interesting. let's go with it for now.